java/src/main/java/com/company/automation/classes/main_task/StudentView.java

```diff
@@ -0,0 +1,81 @@
+package com.company.automation.classes.main_task;
+
+public class StudentView {
+    private final StudentLogic studentLogic = StudentLogic.getStudentLogic();
+    private static final StudentView studentView = new StudentView();
+
+    private StudentView() {
+    }
+
+    public static StudentView getStudentView() {
+        return studentView;
+    }
```

Comment:
The same as mentioned before about singleton

Reply:
Fixed